### ISSUE TITLE
Testng result listening Job

### ISSUE THREAD

**Issue by reply2sesh** [cbeust/testng-eclipse#352]
### Problem Statement
testng hangs with mentioning "Testng result listening job" and never resolves. I had cleaned workspace, removed all related files, Eclipse and installed but still the sam error. even simple program gets hanged. 

Like if you see the screenshot attached, I created new work space with new simple project with just 2 lines and still facing the issue.
 
### The Dependency Management tool for your project
- Maven
- [ ] Eclipse Buildpath: /Library/Java/JavaVirtualMachines/jdk1.8.0_131.jdk/Contents/Home/jre/lib

### Operating System: MAC 10.9.5
![screen shot 2017-08-23 at 3 11 38 pm](https://user-images.githubusercontent.com/11484193/29642469-eb8a83a4-881d-11e7-9070-834fbc2d8c88.png)

Screenshot attached

**Comment by missedone** [comment]
looks like it's trying to setup socket connection between runtime TestNG process and IDE, could you check if there's any antivirus or firewall could impact this? also confirm that there's a `localhost 127.0.0.1` entry in your /etc/hosts

**Comment by reply2sesh** [comment]
Yes there is localhost 127.0.0.1 entry..

**Comment by reply2sesh** [comment]
With Debug mode, I see the below console output

[RemoteTestNG] revisions:

git.commit.id=28d46f3

git.branch=28d46f3138190e289b3865041c86853f39fcf052

git.build.version=1.2.1

[RemoteTestNG] detected TestNG version 6.11.0

[RemoteTestNG] Invoked with -serport 57492 -protocol json -d
/Users/srbomma/eclipseoxygen-workspace/sample/test-output
/var/folders/z0/c8j33f5d6xj_22vd0rtv2z7xw1j24c/T/testng-eclipse-1334428033/testng-customsuite.xml


[BaseMessageSender] Waiting for Eclipse client on localhost:57492

**Comment by missedone** [comment]
pls check around your network settings. (e.g. antivirus, firewall, web proxy, etc.)

**Comment by reply2sesh** [comment]
Verified.They are off.

**Comment by reply2sesh** [comment]
From what I see & understand. when eclipse is opened & it started testng on
port 57354, Then the test is triggered and waiting on different port 61599.

Any thoughts?

2014873740 15421 15416   0  9:58PM ??         0:05.93
/Library/Java/JavaVirtualMachines/jdk1.8.0_131.jdk/Contents/Home/bin/java
-ea -Dfile.encoding=UTF-8 -classpath
/Users/srbomma/.p2/pool/plugins/org.testng.eclipse_6.11.0.201703011520/lib/testng-remote.jar:/Users/srbomma/eclipseoxygen-workspace/sample/target/test-classes:/Users/srbomma/eclipseoxygen-workspace/sample/target/classes:/Users/srbomma/.p2/pool/plugins/org.testng.eclipse_6.11.0.201703011520/lib/testng.jar:/Users/srbomma/.p2/pool/plugins/org.testng.eclipse_6.11.0.201703011520/lib/jcommander.jar:/Users/srbomma/.p2/pool/plugins/org.testng.eclipse_6.11.0.201703011520/lib/bsh-2.0b4.jar:/Users/srbomma/.p2/pool/plugins/org.testng.eclipse_6.11.0.201703011520/lib/snakeyaml.jar:/Users/srbomma/.m2/repository/junit/junit/3.8.1/junit-3.8.1.jar
org.testng.remote.RemoteTestNG -serport 57354 -protocol json -d
/Users/srbomma/eclipseoxygen-workspace/sample/test-output
/var/folders/z0/c8j33f5d6xj_22vd0rtv2z7xw1j24c/T/testng-eclipse-685449704/testng-customsuite.xml


2014873740 17476 15416   0 10:12AM ??         0:00.29
/Library/Java/JavaVirtualMachines/jdk1.8.0_131.jdk/Contents/Home/bin/java
-ea -Dtestng.eclipse.verbose=true -Dtestng.eclipse.debug=true
-Djava.net.preferIPv4Stack=true -Dfile.encoding=UTF-8 -classpath
/Users/srbomma/.p2/pool/plugins/org.testng.eclipse_6.11.0.201703011520/lib/testng-remote.jar:/Users/srbomma/eclipseoxygen-workspace/sample/target/test-classes:/Users/srbomma/eclipseoxygen-workspace/sample/target/classes:/Users/srbomma/.p2/pool/plugins/org.testng.eclipse_6.11.0.201703011520/lib/testng.jar:/Users/srbomma/.p2/pool/plugins/org.testng.eclipse_6.11.0.201703011520/lib/jcommander.jar:/Users/srbomma/.p2/pool/plugins/org.testng.eclipse_6.11.0.201703011520/lib/bsh-2.0b4.jar:/Users/srbomma/.p2/pool/plugins/org.testng.eclipse_6.11.0.201703011520/lib/snakeyaml.jar:/Users/srbomma/.m2/repository/junit/junit/3.8.1/junit-3.8.1.jar
org.testng.remote.RemoteTestNG -serport 61599 -protocol json -d
/Users/srbomma/eclipseoxygen-workspace/sample/test-output
/var/folders/z0/c8j33f5d6xj_22vd0rtv2z7xw1j24c/T/testng-eclipse--1528119182/testng-customsuite.xml

On Thu, Aug 24, 2017 at 10:01 AM, Seshagiri Bommadevara <mail2sesh@gmail.com

**Comment by missedone** [comment]
the log you attached indicates you ran two testng processes..., so there is two port number.

**Comment by missedone** [comment]
closing it since no response, @reply2sesh , feel free to reopen it you still have the issue

**Comment by reply2sesh** [comment]
The issue is intermittent. it works fine and all of the sudden I see issue again.

[RemoteTestNG] revisions:
	git.commit.id=e264a9f
	git.branch=e264a9f5abce10396d0cc41f122682dc6ff9919a
	git.build.version=1.3.0
[RemoteTestNG] loaded class org.testng.internal.Version at file:/Users/srbomma/.m2/repository/org/testng/testng/6.11/testng-6.11.jar
[RemoteTestNG] detected TestNG version 6.11.0
[RemoteTestNG] Invoked with -serport 52239 -protocol json -d /Users/srbomma/eclipseoxygen-workspace/qe/test-output /var/folders/z0/c8j33f5d6xj_22vd0rtv2z7xw1j24c/T/testng-eclipse-1256538135/testng-customsuite.xml 
[BaseMessageSender] Waiting for Eclipse client on localhost:52239

**Comment by LittleFreshMeet** [comment]
@reply2sesh Hi, I also met this issue  recently. Have you find a way to solve it?

**Comment by reply2sesh** [comment]
Nope, I was unable, I had reinstall everything and fortunately it worked..

**Comment by LittleFreshMeet** [comment]
Reinstalling everything is OK for me

**Comment by sriaman3** [comment]
I am getting the same issue-
![image](https://user-images.githubusercontent.com/35685476/83377833-7ba54d80-a3f4-11ea-8afe-33b651732382.png)

**Comment by missedone** [comment]
@sriaman3 , could try with adding jvm arg: `-Djava.net.preferIPv4Stack=true`

**Comment by sriaman3** [comment]
@missedone it is working fine, Thank you for your help.
Could you please brief why we did that?

**Comment by RahulKesyar** [comment]
I'm facing the same problem, (OS is windows). I even tried reinstalling everything but it still didn't work.

can anyone please help me



![2022-06-15_14h23_50](https://user-images.githubusercontent.com/106644747/173786172-a4694d1b-e5e4-4fb2-a7cb-a13f54a737c8.png)
